activeactive/skills/README.md

```diff
@@ -0,0 +1,21 @@
+# Script for manual backup and restore of skills in OnPrem
+
+With skills being present in more than one cluster in Active-Active setup, there is a need to manually backup and 
+restore skills when needed.
+
+## Steps to restore skills on secondary
+
+* Find the skill id to restore. <skill-id>
+* Run script skill-backup.sh in the primary and give skill-id as input.
+  * Example:
+            [root@server0 testadmin]# sh skill-backup.sh
+            Enter skill id:
+             083a05ac-1c1e-4db5-850b-ca1d9d4e56cd
+* This will generate a dir with all the artifacts of that skill.
+* Copy the dir in secondary.
+* Run script skill-restore.sh and give path of the copied folder as input.
+  * Example: 
+             [root@server0 testadmin]# sh skill-restore.sh
+             Enter backup dir path:
+              083a05ac-skill-backup
+
```

Comment:
Please add prerequistes to readme. 
1. Dependencies like `jq`
2. Min AICenter version (22.10)

Reply:
Also mention it requires port forwarding to work.

Reply:
Also seeing other commands like `curl`, `netstat`. Please add those to prerequisites as well.